app/controllers/ops_controller/settings/common.rb

```diff
@@ -481,9 +482,9 @@ def settings_update_save
             end
           end
         else
-          @update.save                                              # Save other settings for current server
+          MiqServer.my_server.settings = @update
```

Comment:
I know it's hard to find all of them, but I would prefer using `MiqServer.my_server.add_settings_for_resource(@update)` here as `#settings=` is just an extra layer of redirection that we don't need.